src/main/java/ToDo.java

```diff
@@ -0,0 +1,15 @@
+public class ToDo extends Task{
+
+    public ToDo(String name){
+        super(name);
+    }
+
+    @Override
+    public void setListName(){
+        if(!isDone){
+            this.listName = "[T]" + this.unmarkedStatus + this.taskName;
+        }else{
+            this.listName = "[T]" + this.markedStatus + this.taskName;
+        }
```

Comment:
Refer to [if-else standard layout](https://se-education.org/guides/conventions/java/basic.html#conditionals).